usb/ehci/phy-ia32-generic.c

```diff
@@ -0,0 +1,175 @@
+/*
+ * Phoenix-RTOS
+ *
+ * EHCI USB Physical Layer for ia32
+ *
+ * Copyright 2024 Phoenix Systems
+ * Author: Adam Greloch
+ *
+ * This file is part of Phoenix-RTOS.
+ *
+ * %LICENSE%
+ */
+
+#include <errno.h>
+#include <stdlib.h>
+#include <string.h>
+#include <sys/mman.h>
+#include <sys/platform.h>
+#include <phoenix/arch/ia32/ia32.h>
+#include <stdio.h>
+#include <sys/platform.h>
+
+#include <hcd.h>
+
+
+#define EHCI_MAP_SIZE (0x1000)
+#define EHCI_PROGIF   (0x20)
+
+
+int hcd_getInfo(const hcd_info_t **info)
+{
+	platformctl_t pctl;
+	int i, err;
+
+	hcd_info_t *hcd_info = NULL;
+
+	pctl.action = pctl_get;
+	pctl.type = pctl_pci;
+	pctl.pci.id.cl = 0x0c03;
```

Comment:
What is this cl/class?

Reply:
Serial bus controller class. Added a proper define